scripts/a2_collections

```diff
@@ -125,15 +130,116 @@ authLoad() {
       # v1 resources work by just re-using output of authQuery except for users, which need a password added
       # TODO: check that all v2 resources work
       if [[ "$RESOURCE" == "users" ]]; then
-        line=$(jq -c '. += {"password": "chefautomate"}' <<< $line)
+        line=$(jq -c '. += {"password": "chefautomate"}' <<< "$line")
       fi
       if [[ "$show_cmd" == 1 ]] || [[ "$dry_run" == 1 ]]; then 
         echo "curl -sSkH \"api-token: \$token\" \"$host/$auth_path/$RESOURCE\" -d '$line'"
       fi
       if [[ "$dry_run" == "" ]]; then 
-        curl -sSkH "api-token: $token" $host/$auth_path/$RESOURCE -d "$line"
+        curl -sSkH "api-token: $token" "$host/$auth_path/$RESOURCE" -d "$line"
         echo
       fi
     fi
   done
 }
+
+#############
+# authGen
+#############
+# Purpose:
+#     Generate a list of a2 auth resources.
+#     This generates dummy data that is not particularly realistic
+#     but can be used to observe general performance characteristics.
+#     NB: When generating policies, it builds one policy per token,
+#     so you should also generate a set of tokens with the same parameters.
+#     This function also allows you to delete your generated resources just as easily.
+# 
+# Example:
+#     Create 500 policies starting with a seed of 101:
+#         $ authGen policies create 500 101
+#     Per above note, those policies need these members:
+#         $ authGen tokens create 500 101
+#     Now, just for illustration, delete the last 100 of those 500 policies:
+#         $ authGen policies delete 100 501
+#
+# Options:
+#    -2 or --v2           use IAM V2 endpoints
+#
+# TODO: Except for common things (e.g. teams) this supports only IAM v1 so far.
+#
+authGen() {
+  # process options
+  auth_path="api/v0/auth"
+  while [[ "$1" =~ ^- ]]; do
+    case "$1" in
+      -2|--v2) auth_path="apis/iam/v2beta"; shift;;
+      -*) echo "'$1' unknown"; return 1;;
+    esac
+  done
+
+  # prerequisites
+  local token=${TOK:?Need this env variable to contain ChefAutomate admin token}
+  local host=${TARGET_HOST:?Need this env variable to point to your ChefAutomate host}
+  [ -z "$4" ] && { echo "usage: authGen [ <options> ] <resource> <mode> <count> <seed>"; return; }
+  if ! type jo &> /dev/null
+  then
+    echo "Need jo installed"; return;
+  fi
+  RESOURCE=$1
+  MODE=$2
+  COUNT=$3
+  SEED=$4
+  ID_PREFIX=test-$RESOURCE
+  [[ ! $MODE =~ ^(create|delete)$ ]] && { echo "mode must be 'create' or 'delete'"; return; }
+  [[ ! $RESOURCE =~ ^(tokens|users|teams|policies)$ ]] && { echo "resource must be in: tokens, users, teams, policies"; return; }
+
+  operation="${MODE}_resource"
+  echo "$MODE $COUNT $RESOURCE..."
+  for (( i = 0; i < COUNT; i++ )) 
+  do
+    $operation "$token" "$host" $i
+  done
+}
+
+function create_resource {
+  local token=$1
+  local host=$2
+  local id_index=$(($3 + SEED))
+  local id=$ID_PREFIX-$id_index
+  local name="$ID_PREFIX $id_index"
+  local json=""
```

Comment:
[nit] I don't believe we need this initialization, i.e. this should do the trick, too:
```suggestion
  local json
```
but it doesn't hurt, I suppose.